app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/twofactorlogin/TwoFactorLoginViewModel.kt

```diff
@@ -205,11 +205,17 @@ class TwoFactorLoginViewModel @Inject constructor(
             -> {
                 val authUrl = authRepository.twoFactorResponse.twoFactorDuoAuthUrl
                 // The url should not be empty unless the environment is somehow not supported.
-                sendEvent(
-                    event = authUrl
-                        ?.let { TwoFactorLoginEvent.NavigateToDuo(uri = Uri.parse(it)) }
-                        ?: TwoFactorLoginEvent.ShowToast(R.string.error_connecting_with_the_duo_service_use_a_different_two_step_login_method_or_contact_duo_for_assistance.asText()),
-                )
+                authUrl
+                    ?.let {
+                    TwoFactorLoginEvent.NavigateToDuo(uri = Uri.parse(it))
+                } ?: mutableStateFlow.update {
```

Comment:
The chaining should have the elvis on it's own line. Also, looks like the indentation was not updated here.